src/Conversion/ONNXToKrnl/Math/RandomNormalLike.cpp

```diff
@@ -0,0 +1,89 @@
+/*
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+//===-------- RandomNormalLike.cpp - Lowering RandomNormal Op -------------===//
+//
+// Copyright 2019-2021 The IBM Research Authors.
+//
+// =============================================================================
+//
+// This file lowers the ONNX Random Normal Like Operator to Krnl dialect.
+//
+//===----------------------------------------------------------------------===//
+
+#include "src/Conversion/ONNXToKrnl/ONNXToKrnlCommon.hpp"
+#include "src/Dialect/Krnl/KrnlHelper.hpp"
+#include "src/Dialect/ONNX/IndexExpr.hpp"
+#include "src/Dialect/ONNX/MLIRDialectBuilder.hpp"
+#include "src/Dialect/ONNX/ShapeInference/ONNXShapeHelper.hpp"
+
+#include <ctime>
+
+using namespace mlir;
+
+struct ONNXRandomNormalLikeOpLowering : public ConversionPattern {
+  ONNXRandomNormalLikeOpLowering(MLIRContext *ctx)
+      : ConversionPattern(
+            mlir::ONNXRandomNormalLikeOp::getOperationName(), 1, ctx) {}
```

Comment:
Now, after upgrading to latest LLVM level we should pass typeConverter to `ConversionPattern`:
```
 : ConversionPattern(
      typeConverter, mlir::ONNXRandomNormalLikeOp::getOperationName(), 1, ctx) {}